packages/utils/testing/src/events.js

```diff
@@ -25,6 +25,10 @@ class Events {
 
   }
 
+  once(name, cb) {
+    cb();
+  }
```

Comment:
This is already solved in #2318 so we might wanna wait until that has landed first.